compiler/luci-interpreter/src/kernels/TestUtils.cpp

```diff
@@ -51,6 +51,17 @@ Matcher<std::vector<float>> FloatArrayNear(const std::vector<float> &values, flo
   return ElementsAreArray(matchers);
 }
 
+Matcher<std::vector<int32_t>> IntArrayEq(const std::vector<int32_t> &values)
```

Comment:
Seems that `::testing::ElementsAreArray` does exactly the same, why don't use it?